src/backend/main/main.py

```diff
@@ -49,4 +49,7 @@ async def check_server_status(request: Request):
 # Run this file to start the server
 if __name__ == "__main__":
     import uvicorn
-    uvicorn.run(main_app, host="0.0.0.0", port=8000, reload=True)
+    import os
+
+    CHAT_SERVICE_PORT = int(os.environ.get("CHAT_SERVICE_PORT"))
```

Comment:
Make it chat_service_port = int(os

Make it small letters

Reply:
I changed it to lowercase chat_service_port